boards/amd/acp_6_0_adsp/acp_6_0_adsp.yml

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright 2024 AMD
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: acp_6_0_adsp/acp_6_0
+name: AMD ACP6.0  Audio DSP
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+  - xcc
```

Comment:
FWIW, pretty much every other SOF platform is moving towards xt-clang at this point.  The last Cadence toolchain version that supports xcc is now over three years old.